gapic/src/main/com/google/gapid/views/PipelineView.java

```diff
@@ -448,6 +453,15 @@ public void update(ViewerCell cell) {
 
                 super.update(cell);
               }
+
+              @Override
+              public String getToolTipText(Object element) {
+                if (dv != null) {
```

Comment:
I am checking this out of paranoia. Let me know if it's unnecessary, and if I should remove it.

Reply:
It is correct and necessary.